config/environments/production.rb

```diff
@@ -69,7 +69,13 @@
   config.i18n.fallbacks = true
 
   config.paperclip_defaults = {
-      storage: :file
+      storage: :s3,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.